2733.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var findNonMinOrMax = function (nums) {
+  const sortedNums = nums.sort((a, b) => a - b);
```

Comment:
```suggestion
  const sortedNums = [...nums].sort((a, b) => a - b);
```
It's better to avoid side effects